.github/workflows/main.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Pytest with Flake8
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9]
+        numpy_ver: [latest]
+        os: [ubuntu-latest, windows-latest]
+        include:
+          - python-version: 3.7
+            numpy_ver: 1.17
+            os: ubuntu-latest
+          - python-version: 3.7
+            numpy_ver: 1.17
+            os: windows-latest
```

Comment:
Is it necessary to require OS here if you want to run both?

Reply:
Yes.  Matrix gives the product of all options above, while include adds single case elements